avoidance/QuadCopterStopAvoidance.hh

```diff
@@ -26,8 +26,12 @@ class QuadCopterStopAvoidance
     : public CollisionAvoidanceStrategy<MavQuadCopter>
 {
   public:
-    QuadCopterStopAvoidance(std::shared_ptr<MavQuadCopter> quadcopter);
+    QuadCopterStopAvoidance(std::shared_ptr<MavQuadCopter> quadcopter,
+                            double trigger_distance = DBL_MAX);
```

Comment:
I left DBL_MAX because it matches the previous behavior (if there is something no matter where -> stop), which is the safest one can get but not very useful (actually it is since there is also a threshold on the sensor side to ignore objects too far). But we can have something like 5.0m as default, what do you think?